packages/git-extractor/src/routes/github/api.ts

```diff
@@ -243,12 +252,23 @@ export async function getCommitTreeSha(
 
   const response: { data: ICommitResponse } = await axios({
     url,
-    headers: { Authorization: `Bearer ${token}` },
+    ...createAxiosRequestConfig(token)
   });
 
   return response.data.commit.tree.sha;
 }
 
+export async function getLatestCommitShaOfFile(username: string, repo: string, branch: string, path: string, token?: string) {
+  const url = buildCommitsUrl(username, repo, branch, path);
+
+  const response: { data: { files: { sha: string }[] } } = await axios({
```

Comment:
This can be simplified like

```suggestion
  const { data: { files: [{ sha }] } } = await axios({
```

Reply:
Consider this being clever about syntax rather than being explicit about what is going on 😉 

`response.data.files[0].sha` reads better than `{ data: { files: [{ sha}] } }`... do not understand why they are introducing all this syntax into the language, just so much more to learn and be opinionated about 😂